controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -144,15 +144,15 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			saved.Status.Phase = status.PhaseProgressing
 		})
 		if err != nil {
-			r.Log.Error(err, "Failed to add conditions to status of DSCInitialization resource.", "DSCInitialization Request.Name", req.Name)
+			r.Log.Error(err, "Failed to add conditions to status of DSCInitialization resource.", "DSCInitialization", req.Namespace, "Request.Name", req.Name)
```

Comment:
Unrelated change, btw

Reply:
yeah, when removed the duplicated one, saw it has better logging than the kept one, so i just use it instead